openlcsd/celery.py

```diff
@@ -8,6 +10,9 @@
 
 app.config_from_object(celeryconfig)
 
+# to access Django settings
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openlcs.openlcs.settings')
```

Comment:
Note that this is not going to work as expected, you can only access settings from hub, not worker.